app-modules/form/src/Models/FormEmailAutoReply.php

```diff
@@ -86,6 +86,7 @@ public function getMergeData(Student|Prospect|null $author): array
             'student last name' => $author->getAttribute($author->displayLastNameKey()),
             'student full name' => $author->getAttribute($author->displayNameKey()),
             'student email' => $author->getAttribute($author->displayEmailKey()),
+            'student preferred name' => $this->getAttribute($author->displayPreferredNameKey()),
```

Comment:
Are you sure this is correct here? The other items seem to reference `$author` not `$this` here?

Reply:
I fixed it so we can get this merged in for deployment.